### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
## Pull request type
 
select from below

- Bug fix
- Documentation

## Which ticket is resolved?

- issue #??

## What does this PR change?
- Fix typos found via `typos --hidden --format brief`

### CODE REVIEW

**Review by MandeepSinghPB08 — APPROVED** [review]

**Comment by miurahr** [comment]
Thank you! but please make test passed with tweaks.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Comment by miurahr** [comment]
The typo fix  of field name borned a zombie of the dead code, that cause behavior change resulted as bug.
A correction of the bug is commited in the master branch

https://github.com/miurahr/py7zr/commit/05edcc236fa4d9154540d030011d717beb29169a
@kianmeng 

A PR said "docs" change, but the PR actually changed the code that has been a potential bug.

File: py7zr/py7zr.py [hunk 1]
```diff
@@ -444,7 +444,7 @@ def _real_get_contents(self, password) -> None:
         header = Header.retrieve(self.fp, buffer, self.afterheader, password)
         if header is None:
             return
-        header._initilized = True
+        header._initialized = True
```
Comment (miurahr) [inline py7zr/py7zr.py:447]: This was the dead code, but the change activate it.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**